src/accordion/accordion.ts

```diff
@@ -33,9 +33,25 @@ let nextId = 0;
   directives: [NgbCollapse]
 })
 export class NgbPanel {
+  /**
+   *  A flag determining whether the panel is disabled or not.
+   *  When disabled, the panel cannot be toggled.
+   */
   @Input() disabled = false;
+
+  /**
+   *  An optional id for the panel. The id should be unique.
```

Comment:
I would add a note: "If not provided by a user it will be auto-generated." or sth along those lines.